src/core/a-animation.js

```diff
@@ -356,13 +357,14 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   var from = {};
   var partialSetAttribute;
   var to = {};
-
-  if (attributeSplit.length === 2) {
+  if (attributeSplit.length === 2 && attributeSplit.indexOf('color') === -1) {
```

Comment:
@ngokevin In this case `attribute = 'color'` so `attrSplit[1]  = undefined`.
The `mapping[attribute] = 'material.color'` so `mapping[attribute].split('.')[0]] = 'material'` aka the component name.